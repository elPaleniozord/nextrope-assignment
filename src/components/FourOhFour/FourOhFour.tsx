import { Button, Container } from "react-bootstrap"

const FourOhFour = () => {
  return (
    <Container>
      <h1>404</h1>
      <h2>Strona nie została odnaleziona</h2>
      <Button href='/'>Powrót na stronę główną</Button>
    </Container>
  )
}

export default FourOhFour