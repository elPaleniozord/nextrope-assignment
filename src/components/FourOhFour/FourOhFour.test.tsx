import React from 'react';
import { render } from '@testing-library/react';
import FourOhFour from './FourOhFour';

test('should render 404 page', () => {
  const { getByText,  asFragment } = render(
    <FourOhFour />
  );

  expect(getByText(/404/)).toBeInTheDocument()
  expect(getByText(/Strona nie została odnaleziona/)).toBeInTheDocument()
  expect(getByText(/Powrót na stronę główną/)).toBeInTheDocument()
  
  expect(asFragment()).toMatchSnapshot()
});
