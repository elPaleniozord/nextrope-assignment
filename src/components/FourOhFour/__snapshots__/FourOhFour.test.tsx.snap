// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 404 page 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h1>
      404
    </h1>
    <h2>
      Strona nie została odnaleziona
    </h2>
    <a
      class="btn btn-primary"
      href="/"
      role="button"
      tabindex="0"
    >
      Powrót na stronę główną
    </a>
  </div>
</DocumentFragment>
`;
